src/components/PathwaysList/PathwaysTable.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, memo } from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEdit, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import {
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow
+} from '@material-ui/core';
+
+import { Pathway } from 'pathways-model';
+import { usePathwayContext } from '../PathwayProvider';
+
+import useStyles from './styles';
+
+const PathwaysTable: FC = () => {
+  const styles = useStyles();
+  const { pathways } = usePathwayContext();
+
+  function deletePathway(pathway: Pathway | null): void {
+    // TODO
+  }
```

Comment:
Since this doesn't maintain a list of all the pathways this should probably be moved to the PathwayProvider